.github/workflows/license.yml

```diff
@@ -44,4 +44,14 @@ jobs:
 
     - name: Check licenses
       # We allow only "notice" type of licenses.
-      run: go run github.com/google/go-licenses check --ignore=golang.org/x --allowed_licenses=Apache-2.0,Apache-3,BSD-3-Clause,BSD-4-Clause,CC0-1.0,ISC,MIT ./...
+        run: |
+          go run github.com/google/go-licenses check \
+            # Ignore our own codebase.
+            --ignore github.com/Jigsaw-Code \
+            --ignore=golang.org/x \
+            # Missing license file, but Apache-2.0. See https://github.com/uber-go/zap/issues/1441#issuecomment-2389189961.
+            --ignore=go.uber.org/zap/exp/zapslog \
+            # TODO: Resolve this indirect Caddy dependency license before releasing the Caddy-backed binary.
+            --ignore github.com/go-sql-driver/mysql \
```

Comment:
Oh, this one is bad. We should file an issue against Caddy. Do you know what is pulling that?

Reply:
Thanks for the explanation. It may be worth it to add some details to the comment.

Consider filing a but directly to github.com/go-sql-driver/mysql as well to ask for them to change their license.

Reply:
I'm concerned that building our main will still pull the harmful dependency.

To prevent it from being pulled, it seems you need to build it with `-tags=nosql`.

Please add a build restriction to the `main.go` to require the `nosql` tag:
```
// +build nosql
```

I'm assuming that the xcaddy command already sets that for the xcaddy build.

I think with that change you will be able to remove the exception here.